projects/packages/my-jetpack/_inc/utils/is-lifetime-purchase.ts

```diff
@@ -1,26 +1,28 @@
+import type { Purchase } from '../data/types';
+
 /**
  * Check if a purchase is considered "Lifetime".
  *
  * @param {object} purchase - A WPCOM purchase object.
```

Comment:
You can actually use typescript types here ✨ 
```suggestion
 * @param {Purchase} purchase - A WPCOM purchase object.
```